environments.json

```diff
@@ -1753,33 +1753,46 @@
     "family": "V8",
     "short": "CH&nbsp;66,<br>OP&nbsp;53",
     "release": "2018-04-17",
-    "retire": "2018-05-31",
+    "retire": "2018-05-29",
     "obsolete": true
   },
   "chrome67": {
     "full": "Chrome",
     "family": "V8",
     "short": "CH&nbsp;67,<br>OP&nbsp;54",
     "release": "2018-05-29",
-    "obsolete": false
+    "retire": "2018-07-24",
+    "obsolete": true
   },
   "chrome68": {
     "full": "Chrome",
     "family": "V8",
     "short": "CH&nbsp;68,<br>OP&nbsp;55",
     "release": "2018-07-24",
+    "retire": "2018-09-04",
```

Comment:
Google Chrome for Android was not released yet...

Reply:
Chrome 68 for Andriod was released in July